### ISSUE TITLE
State Resource XAPI-00221 inconsistency

### ISSUE THREAD

**Issue by Jither** [adlnet/lrs-conformance-test-suite#205]
In State Resource tests:
> An LRS's State Resource can process a GET request with "since" as a parameter (multiplicity, Communication 2.3.s4.table1.row4, XAPI-00221)",

https://github.com/adlnet/lrs-conformance-test-suite/blob/master/test/v1_0_3/H.Communication2.3-StateResource.js#L634

The resulting GET request will have the parameters:

    activityId=http://www.example.com/activityId/hashset
    agent=...
    stateId=1ce36936-06c7-46a2-82ff-a427df3599b5
    since=2017-06-24T07:15:13.567Z

There is no State GET request that allows both a `stateId` and a `since` parameter - it's either/or (Communication#2.3). So it seems to me that according to requirements in Communication#3.2, this particular request should actually be rejected (400) rather than accepted?

> The LRS MUST reject with 400 Bad Request status any requests that use any parameters which the LRS does not recognize in their intended context in this specification. ( Note: LRSs MAY recognize and act on parameters not in this specification).

Also, any otherwise conformant LRS would return a single document, due to the `stateId` parameter, and sure enough, that's also what the test in question expects.

But because the conformance requirement is about the `since` parameter, which only exists in multiple document GET in the first place (that's also the part that "2.3.s4.table1.row4" refers to) - to follow the intent of the conformance requirement, the test should really be expecting an array with the `stateId` it just posted.

In short, it seems to me that this test is both making a request that isn't valid by the specification, and expecting a result that it should never get if the request were valid.
